src/modules/pokemons/PokemonHelper.ts

```diff
@@ -114,6 +114,7 @@ export function incrementPokemonStatistics(pokemonId: number, statistic: Pokemon
     const pokemonStatistics = {
         Captured: App.game.statistics.pokemonCaptured[pokemonId],
         Defeated: App.game.statistics.pokemonDefeated[pokemonId],
+        Seen: App.game.statistics.pokemonSeen[pokemonId],
```

Comment:
Without any gender/shiny/total variants, I don't think it makes sense to have this in this function. Just increment it manually from the RoamerNPC instead.